daffodil-test/src/test/scala/org/apache/daffodil/section00/general/TestImportOtherAnnotationSchema.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.section00.general
+
+import org.junit.Test
+import org.apache.daffodil.tdml.Runner
+import org.junit.AfterClass
+
+object TestImportOtherAnnotationSchema {
+  val testDir = "/org/apache/daffodil/section00/general/"
+  val runner = Runner(testDir, "testImportOtherAnnotationSchema.tdml", validateDFDLSchemas = false)
+
+  @AfterClass def shutDown {
+    runner.reset
+  }
+}
+
+class TestImportOtherAnnotationSchema {
+
+  import TestImportOtherAnnotationSchema._
+
+  //DFDL-1907
+  @Test def test_importOtherAnnotationSchema1() { runner.runOneTest("importOtherAnnotationSchema1") }
+
+}
```

Comment:
Out of curiosity, what happens if a DFDL schema uses types tha tare imported/included from a non-DFDL schema? I assume we would get some kind of "type not found" error since the non-DFDL schema was ignored, thus Daffodil knows nothing about those types? If so, seems like a reasonable behavior since the warning should alert the user about the schema being ignored.

Reply:
Exactly. It will get a missing type - or whatever kind of reference it is. That's the right behavior. Plus the warning about the schema being ignored is a pretty good diagnostic.